tests/urls.py

```diff
@@ -3,15 +3,16 @@
 from rest_framework import routers
 
 from resolwe.api_urls import api_router as resolwe_router
-from resolwe.flow.views import EntityViewSet
-from resolwe_bio.filters import BioEntityFilter
+from resolwe.flow.views import CollectionViewSet, EntityViewSet
+from resolwe_bio.filters import BioCollectionFilter, BioEntityFilter
 from resolwe_bio.kb.views import (
     FeatureViewSet, MappingSearchViewSet
 )
 
 from .routers import SearchRouter
 
 
+CollectionViewSet.filter_class = BioCollectionFilter
```

Comment:
Maybe instead of patching URLs, one could use the `Composer` class. Otherwise, the URLs in the genialis-base repository will also have to be patched.

See https://github.com/genialis/resolwe/blob/90e4eb1160ac3fe0c2f0ac1b6bd18700cd82da55/resolwe/composer.py#L8 

There is some guidance on how to use the composer in

https://github.com/genialis/resolwe/blob/90e4eb1160ac3fe0c2f0ac1b6bd18700cd82da55/resolwe/flow/filters.py#L189

I believe you have to extend the filter used in the `CollectionViewSet` class, which is `resolwe.flow.filters.CollectionFilter`. 

Please try it out and remove line above if it is working.